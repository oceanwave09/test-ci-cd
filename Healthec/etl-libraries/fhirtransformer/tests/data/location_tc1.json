{
  "status": "active",
  "name": "Patient Home",
  "alias": "Home",
  "description": "Patient Home",
  "mode": "kind",
  "type_code": "PTRES",
  "type_system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
  "type_display": "Patient Residence",
  "physical_type_code": "ho",
  "physical_type_display": "House",
  "organization_id": "1c8ed3d8-6bca-47e7-aad4-2c65c7961099"
}