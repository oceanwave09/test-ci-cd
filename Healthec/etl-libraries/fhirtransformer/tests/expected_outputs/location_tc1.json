{
  "resourceType": "Location",
  "status": "active",
  "name": "Patient Home",
  "alias": [
    "Home"
  ],
  "description": "Patient Home",
  "mode": "kind",
  "type": [
    {
      "coding": [
        {
          "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
          "code": "PTRES",
          "display": "Patient Residence"
        }
      ]
    }
  ],
  "physicalType": {
    "coding": [
      {
        "system": "http://terminology.hl7.org/CodeSystem/location-physical-type",
        "code": "ho",
        "display": "House"
      }
    ]
  },
  "managingOrganization": {
    "reference": "Organization/1c8ed3d8-6bca-47e7-aad4-2c65c7961099"
  }
}